ingest-parsed-court-document-event-handler/src/main/scala/uk/gov/nationalarchives/ingestparsedcourtdocumenteventhandler/SeriesMapper.scala

```diff
@@ -6,37 +6,34 @@ import uk.gov.nationalarchives.ingestparsedcourtdocumenteventhandler.SeriesMappe
 import java.net.URI
 
 class SeriesMapper(validCourts: Set[Court]) {
-  def createOutput(
-      metadataPackageLocation: URI,
-      batchId: String,
+  def createDepartmentSeries(
       potentialCourt: Option[String],
       skipSeriesLookup: Boolean
-  ): IO[Output] = {
+  ): IO[DepartmentSeries] = {
     potentialCourt
       .map { court =>
         val potentiallyFoundCourt: Option[Court] = validCourts.find(_.code == court.toUpperCase)
         potentiallyFoundCourt match {
-          case None if skipSeriesLookup => IO.pure(Output(batchId, metadataPackageLocation, None, None))
-          case None                     => IO.raiseError(new Exception(s"Cannot find series and department for court $court for batchId $batchId"))
+          case None if skipSeriesLookup => IO.pure(DepartmentSeries(None, None))
+          case None                     => IO.raiseError(new Exception(s"Cannot find series and department for court $court"))
           case _ =>
             IO.pure(
-              Output(batchId, metadataPackageLocation, potentiallyFoundCourt.map(_.dept), potentiallyFoundCourt.map(_.series))
+              DepartmentSeries(potentiallyFoundCourt.map(_.dept), potentiallyFoundCourt.map(_.series))
             )
         }
       }
-      .getOrElse(IO.pure(Output(batchId, metadataPackageLocation, None, None)))
+      .getOrElse(IO.pure(DepartmentSeries(None, None)))
 
   }
 }
 
 object SeriesMapper {
   case class Output(
       batchId: String,
-      metadataPackage: URI,
-      department: Option[String],
-      series: Option[String]
+      metadataPackage: URI
   )
 
+  case class DepartmentSeries(potentialDepartment: Option[String], potentialSeries: Option[String])
```

Comment:
Maybe `DepartmentAndSeries` instead?

Reply:
I've changed it because we've got a `DepartmentAndSeries` in the ingest-mapper so this will match. Generally I think we should try to move away from "and" in variable names though, it makes them a bit long.